client/my-sites/site-settings/settings-section-header.jsx

```diff
@@ -29,7 +29,7 @@ const SettingsSectionHeader = ( {
 			{ children }
 			{ showButton && (
 				<Button compact primary onClick={ onButtonClick } disabled={ disabled } { ...buttonProps }>
-					{ isSaving ? translate( 'Saving…' ) : translate( 'Save Settings' ) }
+					{ isSaving ? translate( 'Saving…' ) : translate( 'Save settings' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **95** times:
`translate( 'Save Settings' )` *ES Score: 8*